lib/open_project/text_formatting/filters/markdown_filter.rb

```diff
@@ -31,35 +31,36 @@ module Filters
     class MarkdownFilter < HTML::Pipeline::MarkdownFilter
       # Convert Markdown to HTML using CommonMarker
       def call
-        render_html parse
+        render_html
```

Comment:
This method now contains just a single method call. Unless there is a compelling reason, I'd move the implementation of `render_html` into the `call` method.